### PR TITLE
Proofreading

### PR DESCRIPTION
**Pull request by bwilk**
Minor updates in various texts. Typos or grammar corrected

### CODE REVIEW

**Comment by bwilk** [comment]
@kmarszalek please take a look into API docs changes

**Review by kmarszalek — APPROVED** [review]

File: s4e-web/src/app/views/map-view/user-dropdown/user-dropdown.component.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
           target="_blank"
           i18n
         >
-          Zapisz źródło widoku (surowe dane)
+          Metadane i dane surowe
```
Comment (kmarszalek) [inline s4e-web/src/app/views/map-view/user-dropdown/user-dropdown.component.html:66]: Dane surowe i metadane
może brzmieć lepiej?

**Comment by kmarszalek** [comment]
squash commits :)

**Review by Marcelinna — APPROVED** [review]

**Review by abacz — APPROVED** [review]

**Comment by bwilk** [comment]
> squash commits :)

github helps :p

File: s4e-web/src/app/views/map-view/user-dropdown/user-dropdown.component.html (hunk 1 above)
Comment (bwilk) [reply s4e-web/src/app/views/map-view/user-dropdown/user-dropdown.component.html:66]: Missed that. Sorry. The original wording reflects the order in the modal that pops up

**Pull request merged**